tests/python/unittest/test_tir_renew_defs.py

```diff
@@ -169,5 +170,21 @@ def symbolic_func(a: T.handle, b: T.handle, n: T.int32):
     tvm.ir.assert_structural_equal(f1, f2)
 
 
+def test_buffer_map():
+    @T.prim_func
+    def main(a: T.handle, b: T.handle):
+        m = T.int64()
+        A = T.match_buffer(a, (m * 2,))
+        B = T.match_buffer(b, (m, 2))
+        for i, j in T.grid(m, 2):
+            with T.block("B"):
+                vi, vj = T.axis.remap("SS", [i, j])
+                B[vi, vj] = A[vi * 2 + vj]
+
+    f1 = main
+    f2 = tvm.tir.stmt_functor.renew_defs(main)
+    tvm.ir.assert_structural_equal(f1, f2)
```

Comment:
Please check the `m` in `f1` and `f2` are not the same object

Reply:
sg

Reply:
added